docs/api.md

```diff
@@ -367,6 +367,23 @@ documented in the [Browser API ⇗](/docs/browser.md) documentation.
 
 * See [Browser API ⇗](/docs/browser.md)
 
+#### `transport` (Object)
+
+The `transport` option is a shortcut to the [pino.transport()](#pino-transport) function.
+It support the same input options.
+```js
+require('pino')({
+  transport: {
+    target: '/absolute/path/to/my-transport.mjs'
```

Comment:
We also support modules, not just absolute paths.

Reply:
Yes, the transport supports many options and I thought it would be useful a quick example.
Then users may deep down in the `transport` section.

Does it worth adding more examples or ask the user to read the `transport` page?

Reply:
Add a few more examples here.